tutorials/multi/1_graph_classification.py

```diff
@@ -0,0 +1,241 @@
+# ----------------------------------------
+#
+# For communication between multiple processes in multi-gpu training, we need
+# to start the distributed backend at the beginning of each process. We use
+# `world_size` to refer to the number of processes and `rank` to refer to the
+# process ID, which should be an integer from `0` to `world_size - 1`.
+# 
+
+import torch.distributed as dist
+
+def init_process_group(world_size, rank):
+    dist.init_process_group(
+        backend='nccl',
+        init_method='tcp://127.0.0.1:12345',
+        world_size=world_size,
+        rank=rank)
+
+###############################################################################
+# Data Loader Preparation
+# -----------------------
+#
+# Split the dataset into training, validation and test subsets. In dataset
+# splitting, we need to use a same random seed across processes to ensure a
+# same split. We follow the common practice to train with multiple GPUs and
+# evaluate with a single GPU, thus only set `use_ddp` to True in the
+# :func:`~dgl.dataloading.pytorch.GraphDataLoader` for the training set, where 
+# `ddp` stands for :func:`~torch.nn.parallel.DistributedDataParallel`.
+#
+
+from dgl.data import split_dataset
+from dgl.dataloading import GraphDataLoader
+
+def get_dataloaders(dataset, seed, batch_size=32):
+    # Use a 80:10:10 train-val-test split
+    train_set, val_set, test_set = split_dataset(dataset,
+                                                 frac_list=[0.8, 0.1, 0.1],
+                                                 shuffle=True,
+                                                 random_state=seed)
+    train_loader = GraphDataLoader(train_set, use_ddp=True, batch_size=batch_size, shuffle=True)
+    val_loader = GraphDataLoader(val_set, batch_size=batch_size)
+    test_loader = GraphDataLoader(test_set, batch_size=batch_size)
+
+    return train_loader, val_loader, test_loader
+
+###############################################################################
+# Model Initialization
+# --------------------
+#
+# For this tutorial, we use a simplified Graph Isomorphism Network (GIN).
+#
+
+import torch.nn as nn
+import torch.nn.functional as F
+from dgl.nn.pytorch import GINConv, SumPooling
+
+class GIN(nn.Module):
+    def __init__(self, input_size=1, num_classes=2):
+        super(GIN, self).__init__()
+
+        self.conv1 = GINConv(nn.Linear(input_size, num_classes), aggregator_type='sum')
+        self.conv2 = GINConv(nn.Linear(num_classes, num_classes), aggregator_type='sum')
+        self.pool = SumPooling()
+
+    def forward(self, g, feats):
+        feats = self.conv1(g, feats)
+        feats = F.relu(feats)
+        feats = self.conv2(g, feats)
+
+        return self.pool(g, feats)
+
+###############################################################################
+# To ensure same initial model parameters across processes, we need to set a
+# same random seed before model initialization. Once constructed, we wrap the
```

Comment:
if use `constructed` here, the subjective of the sentence should be model?

Reply:
Yes.

Reply:
I mean it should be `once constructed, model blablabla` instead of `once constructed, we blablabla`?

Reply:
Changed to "Once we construct a model instance,"